src/main/java/org/testng/internal/Systematiser.java

```diff
@@ -0,0 +1,96 @@
+package org.testng.internal;
+
+import org.testng.ITestNGMethod;
+
+import java.util.Comparator;
+
+public final class Systematiser {
```

Comment:
If you have some other name in mind please let me know @cbeust I can rename this. I named this so, keeping in mind it helps organize methods in some fashion. But if the name is odd, I can have it named to something like `ComparatorFactory` or something on those lines